.npmignore

```diff
@@ -0,0 +1,44 @@
+# Logs
```

Comment:
But I reinstalled this module yesterday !
Maybe the just haven't published it to npm yet ?
Ok, I'll remove the .npmignore